css/general.css

```diff
@@ -31,17 +31,17 @@
   width: 10px;
   display: inline-block;
 }
-#bootstrap-theme .civicase-panel-empty {
+#bootstrap-theme .civicase__panel-empty {
```

Comment:
What is the use of this `civicase__panel-empty` class? And if you are making changes, this should be moved to scss. We cannot make any changes to the existing css files.